packages/app-store/routing-forms/jsonLogicToPrisma.ts

```diff
@@ -71,13 +88,55 @@ const convertSingleQueryToPrismaWhereClause = (
     logicData[operatorName] instanceof Array ? logicData[operatorName] : [logicData[operatorName]];
 
   const mainOperand = operatorName !== "in" ? operands[0].var : operands[1].var;
+
   let secondaryOperand = staticSecondaryOperand || (operatorName !== "in" ? operands[1] : operands[0]) || "";
   if (operatorName === "all") {
     secondaryOperand = secondaryOperand.in[1];
   }
-  const prismaWhere = {
-    response: { path: [mainOperand, "value"], [`${prismaOperator}`]: secondaryOperand },
-  };
+
+  const isNumberOperator = NumberOperators.includes(operatorName);
+  const secondaryOperandAsNumber = typeof secondaryOperand === "string" ? Number(secondaryOperand) : null;
+
+  let prismaWhere;
+  if (secondaryOperandAsNumber) {
```

Comment:
I rely on `jsonLogicToPrisma.test.ts` here.
Have added 4 new tests for >,>=, <, <=